### PR TITLE
Add Hostwinds

### PR DESCRIPTION
**Pull request by 40504**

### CODE REVIEW

File: entries/h/hostwinds.com.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "Hostwinds": {
+    "domain": "hostwinds.com",
```
Comment (ApCoder123) [inline entries/h/hostwinds.com.json:3]:
```suggestion
    "domain": "hostwinds.com",
    "additional-domains": [
      "hostwinds.co.uk",
      "hostwinds.in",
      "hostwinds.ng",
      "hostwinds.cn",
      "hostwinds.de",
      "hostwinds.es",
      "hostwinds.pt",
      "hostwinds.fr",
      "hostwinds.ru",
      "hostwinds.nl",
      "hostwinds.kr",
      "hostwinds.it",
      "hostwinds.ae"
    ],
```

**Review by ApCoder123 — CHANGES_REQUESTED** [review]
The screenshot provided doesn't really show what types of 2FA are available. Would you be able to take a screenshot of the setup process as evidence? Thanks.

**Comment by 40504** [comment]
image

**Review by ApCoder123 — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]
I think we may need to look at regex support for additional-domains :sweat:

**Pull request merged**